service/src/main/resources/db/migration/R__dev_temp.sql

```diff
@@ -27,31 +29,40 @@ CREATE TABLE vasu_content(
 
 CREATE TRIGGER set_timestamp BEFORE UPDATE ON vasu_content FOR EACH ROW EXECUTE PROCEDURE trigger_refresh_updated();
 
-CREATE TYPE vasu_document_state AS ENUM (
-    'DRAFT',
-    'READY',
-    'REVIEWED',
-    'CLOSED'
-    );
-
 CREATE TABLE vasu_document(
     id uuid PRIMARY KEY DEFAULT ext.uuid_generate_v1mc(),
     created timestamp with time zone DEFAULT now() NOT NULL,
     updated timestamp with time zone DEFAULT now() NOT NULL,
     child_id uuid NOT NULL REFERENCES child(id),
     template_id uuid NOT NULL REFERENCES vasu_template(id),
     content_id uuid NOT NULL REFERENCES vasu_content(id),
-    document_state vasu_document_state NOT NULL,
     modified_at timestamp with time zone NOT NULL,
-    published_at timestamp with time zone,
-    moved_to_ready_at timestamp with time zone,
-    moved_to_reviewed_at timestamp with time zone,
-    vasu_discussion_at timestamp with time zone,
-    evaluation_discussion_at timestamp with time zone
+    vasu_discussion_date date,
+    evaluation_discussion_date date
 );
 
 CREATE TRIGGER set_timestamp BEFORE UPDATE ON vasu_document FOR EACH ROW EXECUTE PROCEDURE trigger_refresh_updated();
 
 CREATE INDEX fk$vasu_document_child_id ON vasu_document(child_id);
 CREATE INDEX fk$vasu_document_template_id ON vasu_document(template_id);
 CREATE INDEX fk$vasu_document_content_id ON vasu_document(content_id);
+
+CREATE TYPE vasu_document_event_type AS ENUM (
+    'PUBLISHED',
+    'MOVED_TO_READY',
+    'RETURNED_TO_READY',
+    'MOVED_TO_REVIEWED',
+    'RETURNED_TO_REVIEWED',
+    'MOVED_TO_CLOSED'
+);
+
+CREATE TABLE vasu_document_event(
+    id uuid PRIMARY KEY DEFAULT ext.uuid_generate_v1mc(),
+    created timestamp with time zone DEFAULT now() NOT NULL,
+    updated timestamp with time zone DEFAULT now() NOT NULL,
+    vasu_document_id uuid NOT NULL REFERENCES vasu_document(id),
+    author_id uuid NOT NULL REFERENCES employee(id),
```

Comment:
`author_id` sounds a bit weird in this context, if it is e.g. the admin who returned it to previous state.